bin/hardening.sh

```diff
@@ -98,6 +99,10 @@ OPTIONS:
         the '-n' option instructs sudo not to prompt for a password.
         Finally note that '--sudo' mode only works for audit mode.
 
+    --set-log-level <level>
```

Comment:
Need to be added to MANUAL.md and README.md
(yes it's not really efficient, you must write it in 3 differents places :disappointed: )